lib/ansible/modules/crypto/openssl_certificate.py

```diff
@@ -299,7 +299,13 @@
         type: bool
         default: no
         version_added: "2.5"
-
+    acme_directory:
+        description:
+            - "The ACME directory to use. You can use any directory that supports the ACME protocol, such as Buypass or Let's Encrypt."
+            - "Let's Encrypt recommends using their staging server while developing jobs. U(https://letsencrypt.org/docs/staging-environment/)."
+        type: str
+        default: https://acme-v02.api.letsencrypt.org/directory
```

Comment:
How about having no default, in which case the default ACME directory of `acme-tiny` is used?

Reply:
I initially thought about that. But it felt hard to communicate that the default is in acme-tiny. Setting it here makes ansible in control over behavior.

Reply:
That's true. But it also forces the user to install at least acme-tiny 4.0.0, because older versions don't support this endpoint (and potentially not the directory parameter, I haven't checked).

On the other hand, Let's Encrypt is in the process of deprecating and disabling ACME v1 anyway, so this should be fine. Could you modify the requirements section to explicitly mention `>= 4.0.0` for acme-tiny?

Reply:
(That should also be mentioned in the changelog fragment.)

Reply:
Thank you. I've updated the PR now. 

I didn't quite understand the numbering system of the change log. So I tried to wing it with an increment. Can you make sure that I've named the change log fragment correctly here?

Reply:
Done, see new comments.